template/template.go

```diff
@@ -286,7 +291,11 @@ func NewTemplateExpander(
 				if math.IsNaN(v) || math.IsInf(v, 0) {
 					return fmt.Sprintf("%.4g", v), nil
 				}
-				t := model.TimeFromUnixNano(int64(v * 1e9)).Time().UTC()
+				timestamp := v * 1e9
+				if timestamp > math.MaxInt64 || timestamp < math.MinInt64 {
+					return "", fmt.Errorf("%T cannot be represented as a nanoseconds timestamp since it overflows int64", v)
```

Comment:
I'd rather print the value here maybe, because %T can only be float

Reply:
yup good catch, fixing